packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeRate.ts

```diff
@@ -139,6 +199,40 @@ export const getTradeRate = async (
     {} as Record<AssetId, ProtocolFee>,
   )
 
+  if (feeAccountInstruction) {
+    if (solAsset) {
+      protocolFees[solAssetId] = {
+        requiresBalance: true,
+        amountCryptoBaseUnit: bnOrZero(protocolFees[solAssetId]?.amountCryptoBaseUnit)
+          .plus(PDA_ACCOUNT_CREATION_COST)
+          .toFixed(),
+        asset: solAsset,
+      }
+    }
+  }
+
+  if (input.sendAddress && solAsset) {
+    const { instruction: createTokenAccountInstruction } = buyAsset
```

Comment:
`buyAsset` is always defined
<img width="722" alt="image" src="https://github.com/user-attachments/assets/8df19bb0-94dd-4e57-936b-077bad7c61cd">

Reply:
I don't get why I added that, should be a mistake, removed in [6840b5c](https://github.com/shapeshift/web/pull/8267/commits/6840b5c3ff9eefd79239117bbd0c17cdb1c4bebf)